src/test/regress/expected/distributed_functions.out

```diff
@@ -618,6 +631,102 @@ SELECT create_distributed_function('eq_with_param_names(macaddr, macaddr)', dist
 
 (1 row)
 
+-- show that we are able to propagate objects with multiple item on address arrays
```

Comment:
this output is hard to understand and could become flaky test. should we add a `WHERE` filter, maybe with the schema name or such? Or, just filter `distributed_object_data`?